core/camel-core-engine/src/main/docs/modules/eips/pages/resume-strategies.adoc

```diff
@@ -0,0 +1,179 @@
+= Resume Strategies
```

Comment:
Is this an EIP? I think this doc should be moved to docs/ where we have the general docs. This dir is only for EIPs

Reply:
Ah silly me yes its implemented as an EIP so the docs location is correct